packages/config/src/load.js

```diff
@@ -66,5 +78,72 @@ export async function loadNuxtConfig ({
     options.rootDir = rootDir
   }
 
+  // Load env to options._env
+  options._env = env
+
+  // Expand and interpolate runtimeConfig from _env
+  for (const c of ['publicRuntimeConfig', 'privateRuntimeConfig']) {
+    if (options[c]) {
+      expand(options[c], options._env)
+    }
+  }
+
   return options
 }
+
+function loadEnv (rootDir) {
+  const env = Object.create(null)
+
+  // Read dotenv
+  const dotenvFile = path.resolve(rootDir, '.env')
```

Comment:
A common approach seem to be to use a different `.env` file for each `NODE_ENV` (or even more like dev, staging, production). Should we support that?

Reply:
As mentioned in the readme, it is both not advised by dotenv and 12 factor. Also, this step is before loading nuxt.config. We can make configurable for loadOptions however :)

Reply:
e809dbfda93f783dbdb640f0cfe06bad79c24993 (not multi-env but path to file can be customized)